### PR TITLE
refactor(core): change script name for core.js

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by Daniellop1 — APPROVED** [review]

**Comment by bartlomieju** [comment]
@nayeemrmn this PR ensures that code from `deno_core` has `deno:` prefix so `PrettyJsError` dims that stack frame. I can't seem to find any test for `deno:` prefix, any ideas?

File: core/runtime.rs [hunk 1]
```diff
@@ -326,7 +326,9 @@ impl JsRuntime {
   fn shared_init(&mut self) {
     if self.needs_init {
       self.needs_init = false;
-      self.execute("core.js", include_str!("core.js")).unwrap();
+      self
+        .execute("deno:core.js", include_str!("core.js"))
```
Comment (nayeemrmn) [inline core/runtime.rs:330]:
```suggestion
        .execute("deno:core/core.js", include_str!("core.js"))
```

**Comment by nayeemrmn** [comment]
> this PR ensures that code from `deno_core` has `deno:` prefix so `PrettyJsError` dims that stack frame. I can't seem to find any test for `deno:` prefix, any ideas?

https://github.com/denoland/deno/pull/7383/files#diff-e1eee92cace256c1f613892877203a4a2d99e66e6d8af7e144e23e8ea5c08aa8R83-R101 adds a test to `op_crates/web/lib.rs` which `isolate.execute()`s some erroneous code which uses one of the bootstrapped functions, and checks that the error string contains the bootstrap script location. Maybe you can do this by calling `Deno.core.jsonOpSync("op_not_registered")` or something.

**Comment by bartlomieju** [comment]
@nayeemrmn thanks, I've added the test, PTAL

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**